modules/alloydb-instance/README.md

```diff
@@ -0,0 +1,87 @@
+# AlloyDB cluster and instance with read replicas
+
+This module manages the creation of AlloyDB cluster and configuration with Automated Backup Policy, Primary node instance and Read Node Pools. 
+
+
+## Simple example
+
+This example shows how to create Alloydb cluster and instance with multiple read pools in GCP project.
+
+```hcl
+module "alloydb" {
+  source               = "./fabric/modules/alloydb-instance"
+  project_id           = "myproject"
+  cluster_id           = "alloydb-cluster-all"
+  cluster_location     = "europe-west2"
```

Comment:
nit: should we use `region` instead to be consistant with the other modules?

Reply:
if the resource attribute is location I would keep that, I seem to remember we do the same for GCS